.identity/env/prod/terraform.tfvars

```diff
@@ -0,0 +1,50 @@
+prefix    = "io"
+env       = "prod"
+env_short = "p"
+domain    = "io-web"
+
+tags = {
+  CreatedBy   = "Terraform"
+  Environment = "Prod"
+  Owner       = "io"
+  Source      = "https://github.com/pagopa/io-web-profile-backend"
+  CostCenter  = "TS310 - PAGAMENTI & SERVIZI"
+}
+
+ci_github_federations = [
+  {
+    repository = "io-web-profile-backend"
+    subject    = "prod-ci"
+  }
+]
+
+cd_github_federations = [
+  {
+    repository = "io-web-profile-backend"
+    subject    = "prod-cd"
+  }
+]
+
+environment_ci_roles = {
+  subscription = ["Reader"]
+  resource_groups = {
+    "terraform-state-rg" = [
+      "Storage Blob Data Reader"
+    ],
+    "dashboards" = [
+      "Reader"
+    ]
+  }
+}
+
+environment_cd_roles = {
+  subscription = ["Reader"]
+  resource_groups = {
+    "terraform-state-rg" = [
+      "Storage Blob Data Contributor"
```

Comment:
Add `Reader and Data Access`

Reply:
done in https://github.com/pagopa/io-web-profile-backend/pull/32/commits/4f8a7668f8d9035acf20e7ef30df8b6cd26ccc3d